scanner/src/lib/common.ts

```diff
@@ -27,6 +27,10 @@ export function formatNumber(input: number, unit = ""): string {
   return result
 }
 
+export function localizeDate(date: string) {
+  return new Intl.DateTimeFormat(locale).format(new Date(date))
```

Comment:
`new Intl.DateTimeFormat(locale)` should be stored in a `const` similar to `NumberFormat` to benefit from the caching as [explained](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleDateString)